Confidence/src/main/java/com/spotify/confidence/DebugLogger.kt

```diff
@@ -0,0 +1,47 @@
+package com.spotify.confidence
+
+import android.util.Log
+
+internal class DebugLogger {
+    var level: DebugLoggerLevel = DebugLoggerLevel.NONE
+
+    internal fun logEvent(tag: String, event: EngineEvent, details: String) {
+        when (level) {
```

Comment:
can we just create a private function like log(message: String) and not repeat this switch case for each log function?